utils/build.config

```diff
@@ -8,7 +8,7 @@ ISAL = v2.30.0
 ISAL_CRYPTO = v2.23.0
 SPDK = v22.01.2
 OFI = v1.15.1
-MERCURY = v2.2.0
+MERCURY = opx-v220
```

Comment:
We know this will need to be updated to match @soumagne 's Mercury version - 2.3?

Reply:
Right likely yes but it might change and I might have to create an rc2 before that.